resources/indexer_make_dag.py

```diff
@@ -49,14 +60,14 @@ def main():
 +FileSystemDomain = "blah"
 should_transfer_files = YES
 transfer_input_files = {os.path.abspath(indexer_script)}
+request_memory = 2000
 notification = Error
 queue
 """)
-        # request_memory = 2500
 
     dagpath = os.path.join(scratch, 'dag')
     with open(dagpath, 'w') as f:
-        for i, path in enumerate(_get_dirpaths()):
+        for i, path in enumerate(_get_dirpaths(args.begin, args.end, args.level)):
             f.write(f'JOB job{i} condor\n')
             f.write(f'VARS job{i} PATH="{path}"\n')
             f.write(f'VARS job{i} JOBNUM="{i}"\n')
```

Comment:
Sorry, I don't know much about Condor or job batches, so I can't help much on if you've got the right format or not.

Reply:
It's right; I've had no problems with what I've been running for L2 data.